src/store/reducer/reducer.js

```diff
@@ -206,8 +204,14 @@ const updateProjectsData = (state, action) => {
       new Date(project.funding_start_year).getFullYear(),
       new Date(project.funding_end_year).getFullYear()
     ];
-    project.collections = typeof project.sammlungen != "undefined" && project.sammlungen[0] != null ? project.sammlungen : ["Keine Sammlung"];
-    project.infrastructure = typeof project.infrastruktur != "undefined" && project.infrastruktur[0] != null ? project.infrastruktur : ["Kein Laborgerät"];
+    project.collections =
+      project.sammlungen && project.sammlungen[0] != null
```

Comment:
project.sammlungen[0] does the same to my knowledge

Reply:
oh ja, stupid me ^^